_example/hello-world/Makefile

```diff
@@ -10,16 +10,16 @@ pack:
 
 dbuild:
 	@docker run --rm \
-		-u ${shell id -u}:${shell id -g} \
```

Comment:
`stat -c` doesn't work here, perhaps one of the differences between POSIX and BSD commands?

`stat -f "%u:%g" .` outputs what we want here: `501:20`

But running `dbuild` gives me me back my original error:
```
> make dbuild
go build -buildmode=c-shared -ldflags="-w -s" -o handler.so
go install _/usr/src/hello-world: open handler.so: permission denied
Makefile:4: recipe for target 'build' failed
make: *** [build] Error 1
make: *** [dbuild] Error 2
```